### PR TITLE
PyPI packaging

### LINKED ISSUE #48: Option to disable offset
I just want to get the latest available image, could it be possible to pass an option to the script so it ignores the offset? 

This instead of having to edit the config file

### LINKED ISSUE #50: Publish package on PyPI
To increase visibility and make installation easier (`pip install himawaripy`), a PyPI package could be created.

To separate code from user configuration, the config file should also be moved outside of the package (I can write a patch for that if you want).

### PR DESCRIPTION
**Pull request by desbma**
- Make all config dynamic by using command line parameters instead of static source file (fixes #48)
- Changes to allow packaging for PyPI (fixes #50)

Once merged, To publish package on PyPI:
- [Create an account](https://pypi.python.org/pypi?%3Aaction=register_form)
- tag version with `git tag -m "Version 1.2.0" 1.2.0 && git push --tags`, this will automatically create a release with a source tarball on GitHub
- Install pandoc `sudo apt-get install pandoc && pip3 install pypandoc`, this allow conversion of the README to RST format used by PyPI
- In source directory, run `python3 setup.py sdist register upload`

Package should then be available on PyPI and installable with `pip install himawaripy`.

### CODE REVIEW

**Comment by boramalper** [comment]
I'm sincerely sorry for being so late, but I have finals coming in three weeks so I cannot spend time on anything else but studying.

After exams, I'll have enough time to spend on my projects, but for now, I can only apologize. Thank you so much!

**Comment by desbma** [comment]
No worries, we all have a life outside of pet projects :)

File: himawaripy/himawaripy.py [hunk 1]
```diff
@@ -106,8 +106,8 @@ def main():
         latest = strptime(loads(latest_json.read().decode("utf-8"))["date"], "%Y-%m-%d %H:%M:%S")
 
     print("Latest version: {} GMT".format(strftime("%Y/%m/%d %H:%M:%S", latest)))
-    if args.hour_offset is not None:
-        requested_time = get_time_offset(latest)
+    if args.hour_offset != 0:
```
Comment (boramalper) [inline himawaripy/himawaripy.py:106]: We can also simply say

``` python
if args.hour_offset:
```

#51

**Comment by boramalper** [comment]
Can you please open separate pull request for each change (not necessarily commit, but you changed a lot of thing and I don't want to merge them all together).

Just one thing: let's keep the configuration file for the backwards compatibility, and make command lines arguments to override. I don't want to break working setups.

Also https://github.com/boramalper/himawaripy/pull/51/commits/a4a85d496e97c428ea97cbd37f3c27a346d2901e#r62180029

Thanks a lot for your hard work, I really appreciate it!

**Comment by desbma** [comment]
> Just one thing: let's keep the configuration file for the backwards compatibility, and make command lines arguments to override. I don't want to break working setups.

Unfortunately this is not a config file, this is a module that is imported. Clean packaging requires that configuration is separated from code that is why I added the command line parameters is the same pull request.

If backwards compatibility is a must for you, you can safely reject this pull request, I won't be mad as I am satisfied with my own changes.

**Comment by boramalper** [comment]
> Unfortunately this is not a config file, this is a module that is imported. Clean packaging requires that configuration is separated from code that is why I added the command line parameters is the same pull request.

I agree, nevertheless I would like to keep it backwards compatible.

**Pull request closed without merging**